ledger/src/blockstore_db.rs

```diff
@@ -1576,6 +1637,11 @@ fn should_exclude_from_compaction(cf_name: &str) -> bool {
     no_compaction_cfs.get(cf_name).is_some()
 }
 
+// Returns true if the column family enables compression.
+fn should_enable_compression<C: 'static + Column + ColumnName>() -> bool {
+    C::NAME == columns::TransactionStatus::NAME
```

Comment:
The PR now only compresses TransactionStatus.  Let me also update the PR title and description as well.